packages/relay-compiler/language/javascript/formatGeneratedModule.js

```diff
@@ -33,12 +36,12 @@ const formatGeneratedModule: FormatModule = ({
 'use strict';
 
 /*::
-import type { ${documentType} } from '${relayRuntimeModule}';
-${flowText || ''}
+${documentTypeImport}
```

Comment:
this would generate incorrect type if `documentTypeImport` is an empty string. In which cases is `documentType` null?

Reply:
As discussed elsewhere, this was basically just following some of the existing code and would indeed generate incorrect code but only if the inputs were already incorrect anyway.